src/types/circle.ts

```diff
@@ -172,6 +172,19 @@ export default /* GraphQL */ `
     targetId: ID!
   }
 
+  input SubscribeCircleInput {
+    "Unique ID."
+    id: ID!
+
+    "Wallet password."
+    password: String!
```

Comment:
Is password required for the first subscription?

Reply:
I'll merge it first, and then we can discuss it in the following works.